src/test/java/com/google/gcp_variant_transforms/library/BigQueryRowGeneratorTest.java

```diff
@@ -0,0 +1,140 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms.library;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.api.services.bigquery.model.TableRow;
+import com.google.common.collect.ImmutableList;
+import com.google.gcp_variant_transforms.TestEnv;
+import com.google.guiceberry.junit4.GuiceBerryRule;
+import com.google.inject.Inject;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.variant.vcf.VCFCodec;
+import htsjdk.variant.vcf.VCFHeader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.IntStream;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+/**
+ * Integration test for generating BigQuery row from VCF record
+ * This test cover the following methods in the Pipeline Runner:
+ * 'generateCodecFromHeaderLines' method in ConvertLineToVariantFn.java
+ * 'getRows' method in ConvertVariantRowFn.java and BigQueryRowGeneratorImpl.java
+ */
+public class BigQueryRowGeneratorTest {
+  private static final String TEST_FILE = "src/test/resources/vcf_files_small_valid-4.0.vcf";
+  private VCFCodec vcfCodec;
+  private VCFHeader vcfHeader;
+  private ImmutableList<String> vcfRecords;
+  private List<TableRow> rows;
+
+  @Rule
+  public final GuiceBerryRule guiceBerry = new GuiceBerryRule(TestEnv.class);
+
+  @Inject
```

Comment:
I think we could make these single lines:

@Inject public VcfParser vcfParser;